sema/subyt/sources.py

```
@@ -284,15 +317,30 @@ class CSVFileSource(Source):
         Source producing iterator over data-set coming from CSV on file
         """
 
-        def __init__(self, csv_file_path: Path) -> None:
+        def __init__(self, csv_file_path: Path, config: dict = {}) -> None:
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)